CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 [jet](https://github.com/borkdude/jet): CLI to transform between JSON, EDN, YAML and Transit using Clojure.
 
+
+## 0.5.26 (2023-06-29)
```

Comment:
Can you change this into "Unreleased" and without a date? I'm going to publish this soon-ish but not yesterday or today since I'm on a vacation.

Reply:
Done -- enjoy your vacation!